pages/blog.tsx

```diff
@@ -94,11 +130,11 @@ export async function getStaticProps({ preview = false }) {
 
   return {
     props: {
-      featuredPost: data.filter((post: Post) => post.featuredPost)[0],
-      secondaryPosts: data.filter((post: Post) => post.secondaryPost),
-      allPosts: data.filter(
-        (post: Post) => !(post.featuredPost || post.secondaryPost) ?? null
-      ),
+      featuredPost: data.filter((post: PostInterface) => post.featuredPost)[0],
+      secondaryPosts: data
+        .filter((post: PostInterface) => post.secondaryPost)
+        .slice(0, 2),
+      allPosts: data,
```

Comment:
This is a question for @stichbury: do we want the `All blog posts` section of the website to include the hero post as well as the two secondary posts?